docs/ids_and_codes/spec_clusters.md

```diff
@@ -1,130 +1,145 @@
+|66            |0x0042    |Proxy Configuration                                       |P  |P  |P    |               |
+|67            |0x0043    |Proxy Discovery                                           |P  |P  |P    |               |
+|68            |0x0044    |Valid Proxies                                             |P  |P  |P    |               |
+|69            |0x0045    |Boolean State                                             |Y  |Y  |Y    |Y              |
+|70            |0x0046    |ICD Management                                            |Y  |Y  |Y    |Y              |
+|72            |0x0048    |Oven Cavity Operational State                             |Y  |Y  |Y    |Y              |
+|73            |0x0049    |Oven Mode                                                 |Y  |Y  |Y    |Y              |
+|74            |0x004A    |Laundry Dryer Controls                                    |Y  |Y  |Y    |Y              |
+|80            |0x0050    |Mode Select                                               |Y  |Y  |Y    |Y              |
+|81            |0x0051    |Laundry Washer Mode                                       |Y  |Y  |Y    |Y              |
+|82            |0x0052    |Refrigerator And Temperature Controlled Cabinet Mode      |Y  |Y  |Y    |Y              |
+|83            |0x0053    |Laundry Washer Controls                                   |Y  |Y  |Y    |Y              |
+|84            |0x0054    |RVC Run Mode                                              |Y  |Y  |Y    |Y              |
+|85            |0x0055    |RVC Clean Mode                                            |Y  |Y  |Y    |Y              |
+|86            |0x0056    |Temperature Control                                       |Y  |Y  |Y    |Y              |
+|87            |0x0057    |Refrigerator Alarm                                        |Y  |Y  |Y    |Y              |
+|89            |0x0059    |Dishwasher Mode                                           |Y  |Y  |Y    |Y              |
+|91            |0x005B    |Air Quality                                               |Y  |Y  |Y    |Y              |
+|92            |0x005C    |Smoke CO Alarm                                            |Y  |Y  |Y    |Y              |
+|93            |0x005D    |Dishwasher Alarm                                          |Y  |Y  |Y    |Y              |
+|94            |0x005E    |Microwave Oven Mode                                       |Y  |Y  |Y    |Y              |
+|95            |0x005F    |Microwave Oven Control                                    |Y  |Y  |Y    |Y              |
+|96            |0x0060    |Operational State                                         |Y  |Y  |Y    |Y              |
+|97            |0x0061    |RVC Operational State                                     |Y  |Y  |Y    |Y              |
+|98            |0x0062    |Scenes Management                                         |Y  |P  |P    |P              |
```

Comment:
I'm aware this is auto generated, but the Y in 1.3 is wrong, right?

Reply:
yeah - will need to re-pull the data model files at 1.3.0.1. Spec tag at 1.3 still has this as non-provisional

Reply:
I'm going to pause this one and do a re-pull of 1.3 before posting this. Thanks, Rene.

Reply:
Fixed now. As is 1.4.2. That took way longer than expected.